Src/KBCsv/CsvReader.cs

```diff
@@ -268,6 +268,30 @@ public char ValueSeparator
         }
 
         /// <summary>
+        /// Gets or sets the character used to override the default line terminators (CR/LF).
+        /// <remarks>
+        /// <para>
+        /// If set to NULL, then the default line terminator (CR/LF) is used.
+        /// </para>
+        /// </remarks>
+        /// </summary>
+        public char? LineTerminatorOverride
```

Comment:
I don't love the naming here. How about just calling it `NewLine` to match the equivalent property in `CsvWriter`. The docs should then make it clear that leaving as `null` will default to treating CR, LF, and CRLF as line endings.